.github/workflows/build-pyodide-package.yml

```diff
@@ -0,0 +1,108 @@
+            arbor_src/ext/random123/include/Random123/features/gccfeatures.h
+        mkdir -p arbor_src/build_arborlib_em
+        cd arbor_src/build_arborlib_em
+        cmake .. \
+            --toolchain "${ABS_TOOLCHAIN}" \
+            -DCMAKE_INSTALL_PREFIX=$(realpath prefix) \
+            -DARB_USE_BUNDLED_LIBS=ON \
+            -DARB_MODCC="${ABS_MODCC}" \
+            -DARB_ARCH=generic
+        make -j 8
+        make install
+    - name: build-arbor-python-so-emscripten
+      run: |
+        ABS_XBUILDENV_INCLUDE=$(realpath xbuildenv/pyodide-root${{ env.CPYTHONINSTALL }}/include/python${{ env.PYMAJOR }}.${{ env.PYMINOR }})
+        ABS_EM_ARBOR=$(realpath arbor_src/build_arborlib_em/prefix)
+        PY_VERSION_FLAT=${{ env.PYMAJOR }}${{ env.PYMINOR }}
+        ARBOR_PYLIB_OUTPUT=arbor/_arbor.cpython-${PY_VERSION_FLAT}-wasm32-emscripten.so
+        ABS_ARBOR_PYLIB_OUTPUT=$(pwd)/$ARBOR_PYLIB_OUTPUT
+        source emsdk-main/emsdk_env.sh
+        mkdir -p arbor arbor.dist-info
+        mkdir -p arbor_src/build_python_lib_em
+        cd arbor_src/build_python_lib_em
+        time em++ \
+            ../python/*.cpp \
+            -I ../ext/pybind11/include/ \
+            -I "${ABS_XBUILDENV_INCLUDE}" \
+            -I "${ABS_EM_ARBOR}/include" \
+            -std=c++20 \
+            -I "${ABS_EM_ARBOR}/lib/libarbor*" \
+            -o "${ABS_ARBOR_PYLIB_OUTPUT}" \
+            -shared -fPIC \
+            -s ALLOW_MEMORY_GROWTH=1 \
+            -s FORCE_FILESYSTEM=1 \
+            -s LINKABLE=1 \
+            -s USE_SDL=0 \
+            -s SIDE_MODULE=1 \
+            -s MODULARIZE=1 \
+            -s WASM_BIGINT=1 \
+            '-DARB_PREFIX="/dev/null"' \
+            '-DARB_BINARY="/dev/null"' \
+            '-DARB_LIB="/dev/null"' \
+            '-DARB_CXX_COMPILER="/dev/null"' \
+            '-DARB_DATA="/dev/null"' \
+            -g0 \
+            -O2
+    - name: build-python-wheel-emscripten
+      run: |
+        cp -v arbor_src/VERSION arbor
+        echo IyBUaGUgUHl0aG9uIHdyYXBwZXIgZ2VuZXJhdGVkIHVzaW5nIHB5YmluZDExIGlzIGEgY29tcGlsZWQgZHluYW1pYyBsaWJyYXJ5LAojIHdpdGggYSBuYW1lIGxpa2UgX2FyYm9yLmNweXRob24tMzgteDg2XzY0LWxpbnV4LWdudS5zbwojCiMgVGhlIGxpYnJhcnkgd2lsbCBiZSBpbnN0YWxsZWQgaW4gdGhlIHNhbWUgcGF0aCBhcyB0aGlzIGZpbGUsIHdoaWNoIHdpbGwgaW1wb3J0cwojIHRoZSBjb21waWxlZCBwYXJ0IG9mIHRoZSB3cmFwcGVyIGZyb20gdGhlIF9hcmJvciBuYW1lc3BhY2UuCgpmcm9tIC5fYXJib3IgaW1wb3J0ICogICMgbm9xYTogRjQwMwoKCiMgUGFyc2UgVkVSU0lPTiBmaWxlIGZvciB0aGUgQXJib3IgdmVyc2lvbiBzdHJpbmcuCmRlZiBnZXRfdmVyc2lvbigpOgogICAgaW1wb3J0IG9zCgogICAgaGVyZSA9IG9zLnBhdGguYWJzcGF0aChvcy5wYXRoLmRpcm5hbWUoX19maWxlX18pKQogICAgd2l0aCBvcGVuKG9zLnBhdGguam9pbihoZXJlLCAiVkVSU0lPTiIpKSBhcyB2ZXJzaW9uX2ZpbGU6CiAgICAgICAgcmV0dXJuIHZlcnNpb25fZmlsZS5yZWFkKCkuc3RyaXAoKQoKCl9fdmVyc2lvbl9fID0gZ2V0X3ZlcnNpb24oKQpfX2NvbmZpZ19fID0gY29uZmlnKCkgICMgbm9xYTpGNDA1CgojIFJlbW92ZSBnZXRfdmVyc2lvbiBmcm9tIGFyYm9yIG1vZHVsZS4KZGVsIGdldF92ZXJzaW9uCg== \
```

Comment:
Clever, but hard to verify and understand. I would either accept these files as files in our git repo, or maybe a separate repo (for now), so that reproducing the wheel does not require the workflow file. We could then hook deployment to a website (eg `arbor-sim.org/online` to that repo) as well. What do you think?

Reply:
Sure!